doc/man/8/ceph-bluestore-tool.rst

```diff
@@ -139,6 +141,10 @@ Commands
    and allows the drive to perform more efficient internal housekeeping.
    If BlueStore runs with discard enabled, this option may not be useful.
 
+:command: `zap` --path *osd path*
```

Comment:
same here

Reply:
by the way, shouldn't it be `zap-device` ?
```suggestion
:command: `zap-device` --path *osd path*
```

Reply:
Fixed.